components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,838 @@
+        result.push({
+            title: 'Invite Members',
+            onClick: () => inviteMembers(sub)
+        })
+        return result;
+    };
+
+    const showCreateTeamModal = () => {
+        const types = getAvailableSubTypes();
+        if (types && types.length > 0) {
+            setCreateTeamModal({ types, defaultCurrency })
+        }
+    }
+
+    const manageMembers = (sub: TeamSubscription) => {
+        setManageTeamModal({ sub });
+    }
+
+    const addMembers = (sub: TeamSubscription) => {
+        setAddMembersModal({ sub });
+    }
+
+    const inviteMembers = (sub: TeamSubscription) => {
+        setInviteMembersModal({ sub });
+    }
+
+    const showBilling = async () => {
+        (await ChargebeeClient.getOrCreate()).openPortal();
+    }
+
+    const isPaymentInProgress = (ts: TeamSubscription) => {
+        return pendingSlotsPurchase && pendingSlotsPurchase.ts.id === ts.id
+    }
+
+    return (<div>
+
+        <div className="flex flex-row">
+            <div className="flex-grow ">
+                <h3 className="self-center">All Teams</h3>
+                <h2>Manage teams and team members.</h2>
+            </div>
+            <div className="flex flex-end space-x-3">
+                {isChargebeeCustomer && (
+                    <button className="self-end my-auto secondary" onClick={() => showBilling()}>Billing</button>
+                )}
+                {getActiveSubs().length > 0 && (
+                    <button className="self-end my-auto" disabled={!!pendingPlanPurchase || getAvailableSubTypes().length === 0} onClick={() => showCreateTeamModal()}>Create Team</button>
+                )}
+            </div>
+        </div>
+
+        {createTeamModal && (
+            <NewTeamModal onClose={() => setCreateTeamModal(undefined)} onBuy={onBuy} {...createTeamModal} />
+        )}
+
+        {manageTeamModal && (
+            <ManageTeamModal onClose={() => { queryState(); setManageTeamModal(undefined) }} slotInputHandler={slotInputHandler} slots={getSlotsForSub(manageTeamModal.sub)} />
+        )}
+
+        {inviteMembersModal && (
+            <InviteMembersModal onClose={() => setInviteMembersModal(undefined)} {...inviteMembersModal} />
+        )}
+
+        {addMembersModal && (
+            <AddMembersModal onClose={() => setAddMembersModal(undefined)} onBuy={onBuy} {...addMembersModal} />
+        )}
+
+        {(getActiveSubs().length === 0 && !pendingPlanPurchase) && (
+            <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100">
+                <div className="m-auto text-center">
+                    <h3 className="self-center text-gray-500 mb-4">No Active Teams</h3>
+                    <div className="text-gray-500 mb-6">Get started by creating a team<br /> and adding team members. <a href="https://www.gitpod.io/docs/teams" target="_blank" rel="noopener" className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Learn more</a></div>
```

Comment:
Nit: The canonical URL has a `/` at the end (this has a minor impact on SEO):

```suggestion
                    <div className="text-gray-500 mb-6">Get started by creating a team<br /> and adding team members. <a href="https://www.gitpod.io/docs/teams/" target="_blank" rel="noopener" className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Learn more</a></div>
```